Tests/Unit/test_complex.py

```diff
@@ -92,20 +117,31 @@ def test_complex_with_multiple_polymers():
     assert str(c3) == str(OrderedComplexSpecies([p[1], p2[0]], called_from_complex = True))
 
 def test_complex_with_polymer_replacement():
-    #This occurs when a Complex is formed around a Monomer in a Polymer in a PolymerConformation which does not include any other Monomers with parents.
+    #These tests show how the order of binding can matter. 
+    #Best practices is to put everything into a PolymerConformation before doing Complex if PolymerConformations are being used.
+
     a = Species('A')
     b = Species('B')
+    s = Species("S")
     p = OrderedPolymerSpecies([a,b,a])
-    pc = Complex([p[0], p[1]]).parent #get a PolymerConformation
+    pc0 = PolymerConformation(polymer = p) #Put the polymer inside a conformation
```

Comment:
why doesn't `Complex` already do that automatically?

Reply:
Complex will automatically make a PolymerConformation from another PolymerConformation. However, it does not make PolymerConformations from Polymers automatically. This is because once in a PolymerConformation, the internal Polymer is immutable. This means that the order Complex is applied matters when mixing PolymerSpecies with PolymerConformations. To avoid this, users who want to use Conformaitons should just place an immutable Polymer inside and use Conformations for everything.